extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -129,15 +132,15 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
         .getVisibilityState();
     const adTestOn = this.element.getAttribute('data-adtest') ||
         isInManualExperiment(this.element);
-    let size;
-    const width = this.element.getAttribute('width');
-    const height = this.element.getAttribute('height');
-    if (width && height) {
-      size = {width, height};
-    } else {
-      size = this.getIntersectionElementLayoutBox();
-    }
-    const format = `${size.width}x${size.height}`;
+    const width = Number(this.element.getAttribute('width'));
+    const height = Number(this.element.getAttribute('height'));
+    // Need to ensure these are numbers since width can be set to 'auto'.
+    // Checking height just in case.
+    this.size_ = isExperimentOn(this.win, 'a4a-use-attr-for-format')
```

Comment:
let's make separate experiments for adsense and doubleclick so we can control independently.